kaolin/transforms/pointcloudfunc.py

```diff
@@ -25,6 +25,41 @@
 EPS = 1e-6
 
 
+def shift(cloud: Union[torch.Tensor, PointCloud],
+          shf: Union[float, int, torch.Tensor]):
+    """Shift the input pointcloud by a shift factor.
+
+    Args:
+        cloud (torch.Tensor or kaolin.rep.PointCloud): pointcloud (ndims >= 2).
+        shf (float, int, torch.Tensor): shift factor (scalar, or tensor).
+        inplace (bool, optional): Bool to make the transform in-place
+    
+    Returns:
+        (torch.Tensor): shifted pointcloud of the same shape as input.
+    
+    Shape:
+        - cloud: :math:`(B x N x D)` (or) :math:`(N x D)`, where :math:`(B)`
+            is the batchsize, :math:`(N)` is the number of points per cloud,
+            and :math:`(D)` is the dimensionality of each cloud.
+        - shf: :math:`(1)` or :math:`(B)`.
+
+    Example:
+        >>> points = torch.rand(1000,3)
+        >>> points2 = shift(points, torch.FloatTensor([3]))
```

Comment:
I would suggest an example that is more explicit about what it is doing, for instance:
```
>>> points = torch.tensor([[0., 0., 0.],
                                          [1., 0., 0.],
                                          [2., 3., 4.]])
>>> shift(points, 3)
tensor([3., 3., 3.],
           [4., 3., 3.],
           [5., 6., 7.]])
```

Reply:
Sure, I'll make the changes!